utilities/src/parallel.rs

```diff
@@ -67,18 +67,18 @@ pub fn max_available_threads() -> usize {
 }
 
 #[inline(always)]
-#[cfg(not(feature = "serial"))]
+#[cfg(not(any(feature = "serial", feature = "wasm")))]
 pub fn execute_with_max_available_threads<T: Sync + Send>(f: impl FnOnce() -> T + Send) -> T {
     execute_with_threads(f, max_available_threads())
 }
 
 #[inline(always)]
-#[cfg(feature = "serial")]
+#[cfg(any(feature = "serial", feature = "wasm"))]
 pub fn execute_with_max_available_threads<T>(f: impl FnOnce() -> T + Send) -> T {
     f()
 }
 
-#[cfg(not(feature = "serial"))]
+#[cfg(not(any(feature = "serial", feature = "wasm")))]
```

Comment:
Outside of wasm, `execute_with_threads` starts up an isolated thread pool to churn through all the rayon annotated paralleism (i.e. most parallel iterators). However within the `wasm` context using `wasm_bindgen_rayon` a global thread pool is spun up using webworkers and the feature flagged version of `execute_with_max_available_threads` above takes will run the function using the available global web-worker thread pool. So this flag isn't actually taking away parallelism in the case of the non-serial wasm flag.